controller/backup_controller.go

```diff
@@ -313,17 +316,17 @@ func (bc *BackupController) reconcile(backupName string) (err error) {
 	backupURL := backupstore.EncodeBackupURL(backup.Name, backupVolumeName, backupTargetClient.URL)
 	backupInfo, err := backupTargetClient.InspectBackupConfig(backupURL)
 	if err != nil {
-		if !strings.Contains(err.Error(), "in progress") {
-			log.WithError(err).Error("Error inspecting backup config")
-		}
+		log.WithError(err).Error("Error inspecting backup config")
 		return nil // Ignore error to prevent enqueue
 	}
+
+	backup.Status.State = longhorn.BackupStateCompleted
+	backup.Status.LastSyncedAt = syncTime
```

Comment:
Any explanation for this modification?

Reply:
I thought that if the backup inspection succeeded (without error) but the backup info in nil, we should consider the backup synced with the remote backup target already. Therefore, updates the backup state to `Completed` and `lastSyncedAt`.

Reply:
How can the backup info be nil? Does this mean something is wrong with the backup config file?

Reply:
In general, it's impossible that the err = nil and backupInfo = nil.

Never mind, I will revert this change.

Reply:
OK. You can print a warning log if `backupInfo == nil` for now.